source/_docs/automation/trigger.markdown

```diff
@@ -263,6 +290,8 @@ automation:
 
 The `for` template(s) will be evaluated when an entity changes as specified.
 
+You may use a [template](/docs/configuration/templating/) for configuring the `entity_id` property. This is useful for Blueprint developers to allow a selector to specify the entities for the trigger. The template will only be evaluated at configuration time.  Only [limited templates](/docs/configuration/templating/#limited-templates) are allowed.
```

Comment:
```suggestion
You may use a [template](/docs/configuration/templating/) for configuring the `entity_id` property. The template will only be evaluated at configuration time. Only [limited templates](/docs/configuration/templating/#limited-templates) are allowed.
```

Reply:
Removed references to blueprint and fixed white space issues.